### PR TITLE
Clean up docs usage

### PR DESCRIPTION
**Pull request by KristofKoch**
**Description of proposed changes**

Changed formatting of usage message as per #5341.

### CODE REVIEW

File: src/docs.c [hunk 1]
```diff
@@ -41,19 +41,25 @@
...
 	if (level == GMT_SYNOPSIS) return (GMT_MODULE_SYNOPSIS);
-	GMT_Message (API, GMT_TIME_NONE, "\t<module-name> is one of the core or supplemental modules, or one of\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   api, colors, cookbook, data, gallery, gmt, home, settings, and tutorial.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Also acceptable are forum (GMT Discourse Forum) and [web]site (GMT Main Website).\n");
+
+	GMT_Message (API, GMT_TIME_NONE, "  REQUIRED ARGUMENTS:\n");
+	GMT_Usage (API, 1, "\n<module-name>");
+	GMT_Usage (API, -2, "One of the core or supplemental modules, or one of "
+		"api, colors, cookbook, data, gallery, gmt, home, settings, and tutorial. "
+		"Also acceptable are forum (GMT Discourse Forum) and [web]site (GMT Main Website).");
```
Comment (meghanrjones) [inline src/docs.c:52]:
```suggestion
		"Also acceptable are forum (GMT Discourse Forum) and website (GMT Main Website).");
```
`gmt docs web` does not work but `gmt docs website` does

**Review by meghanrjones — APPROVED** [review]

File: src/docs.c (hunk 1 above)
Comment (KristofKoch) [reply src/docs.c:52]: Carried over from original. `gmt docs site` works as well. 

Maybe shift the parentheses? -> `web[site]`

Comment (meghanrjones) [reply src/docs.c:52]: > Carried over from original. `gmt docs site` works as well.
> 
> Maybe shift the parentheses? -> `web[site]`

sounds good

Comment (KristofKoch) [reply src/docs.c:52]:
```suggestion
		"Also acceptable are forum (GMT Discourse Forum) and web[site] (GMT Main Website).");
```

**Pull request merged**